bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -5938,6 +6041,189 @@ private static void closeStatement(Statement statement) {
 	}
 
 	/**
+	 * Sets the end date for the given ingest job
+	 *
+	 * @param ingestJobId The ingest job to set the end date for
+	 * @param endDateTime The end date
+	 *
+	 * @throws TskCoreException If inserting into the database fails
+	 */
+	void setIngestJobEndDateTime(long ingestJobId, long endDateTime) throws TskCoreException {
+		CaseDbConnection connection = connections.getConnection();
+		acquireSharedLock();
+		try {
+			Statement statement = connection.createStatement();
+			statement.executeUpdate("UPDATE ingest_jobs SET end_date_time=" + endDateTime + " WHERE ingest_job_id=" + ingestJobId + ";");
+		} catch (SQLException ex) {
+			throw new TskCoreException("Error updating the end date.", ex);
+		} finally {
```

Comment:
It would be nice to include the ingest_job_id in this log message to help with debugging: String.format("Error updating end date (ingest_job_id=&d)", ingestJobId)